src/MartinGeorgiev/Doctrine/ORM/Query/AST/Functions/JsonbGetFieldAsText.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MartinGeorgiev\Doctrine\ORM\Query\AST\Functions;
+
+/**
+ * Implementation of PostgreSql json field retrieval as text, filtered by key (using ->>).
+ *
+ * @see https://www.postgresql.org/docs/9.4/static/functions-json.html
+ * @since 0.1
+ *
+ * @author Martin Georgiev <martin.georgiev@gmail.com>
+ */
+class JsonbGetFieldAsText extends BaseFunction
+{
+    protected function customiseFunction(): void
+    {
+        $this->setFunctionPrototype('(to_jsonb(%s) ->> %s)');
```

Comment:
Here is a similar comment to the one for the proposed changes in `src/MartinGeorgiev/Doctrine/ORM/Query/AST/Functions/JsonGetFieldAsText.php`.